.github/workflows/verify-pr-commit.yml

```diff
@@ -209,8 +209,14 @@ jobs:
     steps:
       - name: Check Out Repo
         uses: actions/checkout@v3
-      - name: Build Rust Docs
-        run: RUSTDOCFLAGS="--enable-index-page --check -Zunstable-options" cargo doc --no-deps --features frequency
+        # cargo doc requires a toolchain version ^nightly-2022-11-15 because of a bug that causes a panic
+        # When the toolchain is updated to ^nightly-2023-06-01, this can be removed
+      - name: Install Rust Nightly-2023-06-01 and Build Docs
+        run: |
+          export HOME="/root"
```

Comment:
I don't know why this `export` was required, but it fixes an issue where rust was complaining about rustup not being installed in `/github/home/.cargo`.

Reply:
I think this is fine for now as a temp fix.

@demisx Any thoughts?